apps/common-lib/src/main/scala/com/gu/typerighter/rules/Dictionary.scala

```diff
@@ -25,7 +26,6 @@ object Dictionary {
       lemmaOrInflList <- lemmaOrInfl.toList
       word <- lemmaOrInflListToText(lemmaOrInflList)
     } yield word
-    words.distinct.filterNot(_ == "")
+    words.distinct.filterNot(_.filterNot(_.isWhitespace) == "")
```

Comment:
Would `words.distinct.filterNot(_.forall(_.isWhitespace))` be more obvious?

Reply:
```suggestion
    words.distinct.filterNot(_.forall(_.isWhitespace))
```

Much nicer, ta!